mep/people/views.py

```diff
@@ -104,8 +112,13 @@ def get_context_data(self, **kwargs):
     def get_page_labels(self, paginator):
         '''generate labels for pagination'''
 
+        # if form is invalid, page labels should show 'N/A'
+        form = self.get_form()
+        if not form.is_valid():
+            return [(1, 'N/A')]
```

Comment:
nice adjustment

Reply:
I didn't see a unit test for this case, did you add one?

Reply:
I did but maybe it landed after your review; it's here:
https://github.com/Princeton-CDH/mep-django/blob/f58874ddcfecebedfa6b35fbeac360e5040ad503/mep/people/tests/test_views.py#L797-L799